tests/src/core/testqgsnetworkaccessmanager.cpp

```diff
@@ -778,7 +769,7 @@ void TestQgsNetworkAccessManager::testAuthRequestHandler()
   QString expectedPassword;
   int requestId = -1;
   QUrl u =  QUrl( QStringLiteral( "http://" ) + mHttpBinHost + QStringLiteral( "/basic-auth/me/" ) + hash );
-  QNetworkReply::NetworkError expectedError = QNetworkReply::AuthenticationRequiredError;
+  QNetworkReply::NetworkError expectedError = QNetworkReply::NoError;
```

Comment:
Was this a mistake?

Reply:
The value of the variable `expectedError`, at this point (at the stage of creating lambda functions), has no meaning. It will be overridden just before the request is sent, a few lines below. I found this very instructive :) Therefore, I decided to roll back this change made by PR #42629